vpplink/rdma.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2020 Cisco Systems Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vpplink
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/projectcalico/vpp-dataplane/vpplink/binapi/vppapi/rdma"
+)
+
+type RDMAInfo struct {
+	HostIf  string
+	RxqNum  int
+	RxqSize int
+	TxqSize int
+}
+
+func (v *VppLink) CreateRDMA(rdmainfo RDMAInfo) (swIfIndex uint32, err error) {
```

Comment:
Might be worth making `rdmainfo` a pointer, also maybe worth using a name like `RDMAInterface`
At some point, we could even try to make the all the Interface structs uniforms (and share a common struct)